### PR TITLE
[OBSDEF-6857] Fixed ts lint issues and warnings

### PR DESCRIPTION
**Pull request by tejasjc-ecs**
## Summary of the changes 
_List the changes from this PR_
[OBSDEF-6857](https://jira.cec.lab.emc.com/browse/OBSDEF-6857)

### Summary
- Fixed linting errors and warnings (Check [OBSDEF-6857](https://jira.cec.lab.emc.com/browse/OBSDEF-6857) description for errors and warnings reference)
- Disabled some linting warning rules as it needed major code changes
- Updated PR validation and release pipeline to run lint checks
- Capitalized wizard action buttons text
- Increase version

image

## Where to test ?
Tested on andre as well.
 
http://10.249.253.4:3007/

```
eyJhbGciOiJSUzI1NiIsImtpZCI6Im9RNE1BLVlFNU5GYi1qZDdrTld1OG9yVmpmZXhHcW9USkJEWWZNM1MxV1UifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJzdmMtb2JqZWN0c2NhbGUtZGV2LWRvbWFpbi1jMjAiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlY3JldC5uYW1lIjoib2JqZWN0c2NhbGUtb3BlcmF0b3ItdG9rZW4tMmd0ODgiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC5uYW1lIjoib2JqZWN0c2NhbGUtb3BlcmF0b3IiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlcnZpY2UtYWNjb3VudC51aWQiOiI5MzU3M2JhNy02MWJmLTRhZmMtOGJjNS1lOTQzMTFjN2UxYTIiLCJzdWIiOiJzeXN0ZW06c2VydmljZWFjY291bnQ6c3ZjLW9iamVjdHNjYWxlLWRldi1kb21haW4tYzIwOm9iamVjdHNjYWxlLW9wZXJhdG9yIn0.89-9R7y1Rhsok99xg16yk4LK_dHVcF6c1yY0qCdnM_dxpFIJg7UuWYJvw_4VhODU51pjuALlgDQl-680fBoVsaJTWQSB-Upblf4upoLQvxYvcu_gHUKyuzla--vJ84RTvtt6MBCC9L8tbAE9jt0rDP9S1-WJSIRk4yo9MP6fWmKkGcvk5bAK_UbGE666wLzCeKXiSTx5P4_bp6rpX8eZFSynHbWoV3xNmO4oDJMJ6BO0k4OVmxNq5jOTB_2Vb1inhNt7Mbq7pUe8xwnltHhIK_soRC4HSgHe3AZuITsI38GV-NAGV7pJ_6A0jy2ozx0HPmBfFnnss6780NM6DD32Wg
```

In case you suspect any issue, cross-check on existing env below if it exists there as well in that way it will be efficient to track down the issue
> kubeadmin  JbRur-Jca8w-t8GtU-sTtxy
https://10.236.68.48:4443/

## Verified on Platform(s)
_Select the platforms on which you have verified the changes_
- [x] vSphere plugin
- [x] OpenShift
- [x] Standalone

## Peer Tester Name
Check Tabs, Wizards and datagrids filter, sorting and other actions.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@dellstorage/clarity-react",
-    "version": "1.1.20",
+    "version": "1.1.21",
```
Comment (maheshkulkarni4) [inline package.json:3]: Good time to move 1.2.0 ?

File: src/cards/Cards.stories.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ storiesOf("Cards", module)
                 <div className="clr-col-lg-4 clr-col-md-8 clr-col-12">
                     <Card onClick={action("onclick - card image top")}>
                         <CardImage>
-                            <img src={require("./placeholder_350x150.png")} />
+                            <img src={require("./placeholder_350x150.png")} alt="" />
```
Comment (maheshkulkarni4) [inline src/cards/Cards.stories.tsx:49]: Cant we add a rule to exclude this change.  No need to add ALTs at every image tag.

**Review by maheshkulkarni4 — APPROVED** [review]
Changes look good, can you please upload the Lint report on the PR too? 
Please get this tested with VSphere Plugin Just to make sure all is good.

File: package.json (hunk 1 above)
Comment (tejasjc-ecs) [reply package.json]: @maheshkulkarni4 @akate1 started with version 1.2.0 now

File: src/cards/Cards.stories.tsx (hunk 2 above)
Comment (tejasjc-ecs) [reply src/cards/Cards.stories.tsx]: Disabled alt warning rule now

**Review by Malayvarma27 — APPROVED** [review]
Tested: 
1- Datagrid: filtering, show/hide columns, sorting, pagination
2- Account details tabs
3- Signpost
4- Wizard steps

**Review by akate1 — APPROVED** [review]

**Pull request merged**